app/models/katello/content_view_environment.rb

```diff
@@ -75,16 +68,16 @@ def priority(content_object)
       end
     end
 
-    def self.fetch_content_view_environments(labels: [], ids: [], organization:)
+    def self.fetch_content_view_environments(candlepin_names: [], ids: [], organization:)
```

Comment:
if labels and candlepin_names are synonymous why change this ?

Reply:
I think it was left over from before I discovered that they were the same. I can change it back